storm-core/src/jvm/org/apache/storm/grouping/ShuffleGrouping.java

```diff
@@ -17,17 +17,13 @@
  */
 package org.apache.storm.grouping;
 
-import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
-
 import org.apache.storm.generated.GlobalStreamId;
 import org.apache.storm.task.WorkerTopologyContext;
 
+import java.io.Serializable;
+import java.util.*;
```

Comment:
nit: we usually discourage widcard imports.  They don't always maintain compatibility.